docs/src/main/asciidoc/funqy-gcp-functions.adoc

```diff
@@ -40,6 +40,34 @@ At the time of this writing, Cloud Functions are still in beta so make sure to i
 gcloud components install beta
 ----
 
+== The Quickstart
+
+Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+
+The solution is located in the `funqy-google-cloud-functions-quickstart` {quickstarts-tree-url}/funqy-quickstarts/funqy-google-cloud-functions-quickstart[directory].
+
+== Creating the Maven Deployment Project
+
+Create an application with the `quarkus-funqy-google-cloud-functions` extension.
+You can use the following Maven command to create it:
+
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=funqy-google-cloud-functions \
+    -DclassName="org.acme.quickstart.GreetingResource" \
+    -Dpath="/hello" \
+    -Dextensions="funqy-google-cloud-functions"
+----
+
+Now, let's remove what's not needed inside the generated application:
+
+- Remove the dependency `io.quarkus:quarkus-reasteasy` from your `pom.xml` file.
+- Remove the generated `org.acme.quickstart.GreetingResource` class.
+- Remove the `index.html` from `resources/META-INF/resources` or it will be picked up instead of your Function.
+- Remove the existing tests.
+
```

Comment:
Looks like codestarts will help out here, right? 

@ia3andy @patriot1burke

Reply:
Yes, some day ...

Reply:
@machi1990  Yes, you will just need to add `skipExample`.



> Yes, some day ...

@loicmathieu it shouldn't be that long before we make it default, probably 1.8.x @maxandersen?

Reply:
Thanks for the response. Can't wait till this becomes the default, as this paragraph in doc just shows the need for it.

Reply:
@ia3andy I means some day I will take a look at codestart and implement them for my extensions. 
I just didn't have that time in the near future